core/services/mongodb-remote/src/test/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizerUnitTests.kt

```diff
@@ -832,7 +943,212 @@ class DataSynchronizerUnitTests {
     }
 
     @Test
-    fun testDeleteOneById() {
+    fun testUpsertOne() {
```

Comment:
[nit] one other test that might be worth adding is to do an update where upsert is specified, but there are documents that match, and we want to verify that no insert occurs

Reply:
I believe that's covered here– but I can add another check to make it clearer.